src/websocket-client.ts

```diff
@@ -296,8 +296,10 @@ export class WebsocketClient extends EventEmitter {
       }
 
       // Binance allows unsolicited pongs, so we send both (though we expect a pong in response to our ping if the connection is still alive)
-      ws.ping();
-      ws.pong();
+      if (ws.readyState === 1) {
+        ws.ping();
+        ws.pong();
+      }
```

Comment:
Just in case there's a troubleshooting need in future:

```suggestion
      if (ws.readyState === 1) {
        ws.ping();
        ws.pong();
      } else {
        this.logger.silly(
          `WS ready state not open - refusing to send WS ping`,
          { ...loggerCategory, wsKey, readyState: ws?.readyState },
        );
      }
```

Reply:
Yeah, it makes sense.

I can also see a case if/when the WebSocket gets stuck in state different than OPEN (not sure if it could ever happen, but if...), pongTimeout never occurs (since it is cleared before `tryWsPing()` is called), and there would be no indication of any problem.